code/modules/antagonists/hivemind/hivemind.dm

```diff
@@ -184,4 +205,4 @@
 	return result.Join("<br>")
 
 /datum/antagonist/hivemind/is_gamemode_hero()
-	return SSticker.mode.name == "hivemind"
+	return SSticker.mode.name == "Assimilation"
```

Comment:
Consistent naming reeee

Reply:
just fixing the function here, i'll look into removing hivemind as the interal/config name some other time